docs/circuit-test.md

```diff
@@ -44,6 +44,49 @@ View a history of completed circuit tests, including the provider the call route
 + **Filters** pop out on the right allowing you to filter some of your data.
 + Adjust the Column ordering.
 
+|Parameter|Description|
+|---------|-----------|
+|**Created**| When the log was created mm/dd/yyyy|
+|**Provider**|List of your providers|
+|**MOS- Mean Opinion Score**|To find the call quality|
+|**DTMF- Dual Tone Multi Frequency**|diagnose and resolve various communication problems, enhancing network performance and ensuring smooth communication experiences|
+|**Answered**|Call has been connected and answered|
+|**Destroyed**|Call has been killed|
+|**Direction**|Either **Termination** or **Origination**|
+|**Card**|Displays Card-ID/Name|
+|**Destination Number**|Recipient's number |
+|**Caller**|1. **ID**: Caller's ID number <br> 2. **Display Name** of the caller <br> 3. **IP**: IP address of the caller|
+|**Called**|1. **ID**: Recipient's ID number <br> 2. **Display Name** of the recipient <br> 3. **IP**: IP address of the recipient|
+|**SIP Call ID**|SIP call ID number|
+|**Disconnection**|1. **Code**: standardized code used to identify the reason why a call was disconnected. <br> 2. **Status**: Specific reason why a call was terminated unexpectedly. <br> These **codes** and **status **are crucial for troubleshooting issues, analyzing network performance, and understanding call quality.|
+|**Released By**|BYE message sent by- Upstream or Downstream |
```

Comment:
Indicates whether the call termination was initiated by Upstream or Downstream.

Reply:
done